docs/src/pages/index.tsx

```diff
@@ -4,24 +4,44 @@
  */
 
 import Layout from "@theme/Layout";
-import Cookies from "js-cookie";
 import React from "react";
 import { v4 as uuidv4 } from "uuid";
 
 import appInsights from "@site/src/appInsights";
 import { Homepage } from "@site/src/components/home";
 
+/**
+ * Helper function to get a cookie value
+ * @param name - The name of the cookie
+ * @returns - The value of the cookie
+ */
+function getCookie(name): string | undefined {
+	const match = document.cookie.match(new RegExp(`(^| )${name}=([^;]+)`));
+	return match ? decodeURIComponent(match[2]) : undefined;
+}
+
+/**
+ * Helper function to set a cookie value
+ * @param name - The name of the cookie
+ * @param value - The value to set
+ * @param days - The number of days until the cookie expires
+ */
+function setCookie(name, value, days): void {
+	const expires = new Date(Date.now() + days * 86400000).toUTCString();
+	// eslint-disable-next-line unicorn/no-document-cookie
+	document.cookie = `${encodeURIComponent(name)}=${encodeURIComponent(value)}; expires=${expires}; path=/; SameSite=Strict`;
```

Comment:
Interesting, did the original implementation encode the name too? That would cause weird behavior if someone tries to set a cookie with a name that gets converted during encoding. It can never be retrieved because `getCookie` doesn't try to encode its input before matching. I'd say we can remove encoding for the name but also throw at the start of this function if there's anything other than `[a-Z0-9-_]` in the name? It's an internal function just for us and I don't think we'll ever need to handle complicated cookie names.